AirCasting/Settings/ModalSettings.swift

```diff
@@ -0,0 +1,66 @@
+// Created by Lunar on 28/06/2021.
+//
+
+import SwiftUI
+import AirCastingStyling
+
+struct BackendSettingsModalView: View {
```

Comment:
It's usually a common practice to name files after classes/structs/protocols they contain. So here it would be `BackendSettingsModalView.swift`. I also think that the `Modal` part in the name is not needed because the view itself does not have a knowledge on how it's being presented